cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -37,17 +37,20 @@
     public static final String STRIMZI_KAFKA_CONNECT_S2I_IMAGES = "STRIMZI_KAFKA_CONNECT_S2I_IMAGES";
     public static final String STRIMZI_KAFKA_MIRROR_MAKER_IMAGES = "STRIMZI_KAFKA_MIRROR_MAKER_IMAGES";
     public static final String STRIMZI_IMAGE_PULL_POLICY = "STRIMZI_IMAGE_PULL_POLICY";
+    public static final String STRIMZI_FORCE_OPENSHIFT = "STRIMZI_FORCE_OPENSHIFT";
```

Comment:
instead of "force" that let me think we could "force" the cluster to change from Kube to OpenShift which is obviously impossible :-) ... can we use "assume"?

Reply:
I also wondered about force. If we going to change the name then `STRIMZI_IS_ON_OPENSHIFT` is surely the most self-explanatory option.

Reply:
That sounds reasonable. I also thought `FORCE` is wrong, but the right word didn't came to my mind. I will change it.

Reply:
Ok, GitHub actually didn't showed me the comment from @tombentley. I changed it to `ASSUME` because I think that better covers what setting it to false means (it is treated as us assuming we are not on OpenShift). If you want to change it to `STRIMZI_IS_ON_OPENSHIFT` please let me know. - I had it there originally and it didn't fit for me.